console/cli_context.go

```diff
@@ -230,7 +230,7 @@ func (r *CliContext) Secret(question string, option ...console.SecretOption) (st
 }
 
 func (r *CliContext) Spinner(message string, option console.SpinnerOption) error {
-	style := lipgloss.NewStyle().Foreground(lipgloss.Color("#8ED3F9"))
+	style := lipgloss.NewStyle().Foreground(lipgloss.CompleteColor{TrueColor: "#3D8C8D", ANSI256: "30", ANSI: "6"})
```

Comment:
As the description: https://github.com/charmbracelet/lipgloss?tab=readme-ov-file#true-color-16777216-colors-24-bit, the color will be automatically detected by Terminal, it's unexpected. We should use the CompleteColor.